docs/buildah-rmi.md

```diff
@@ -4,7 +4,7 @@
 buildah\-rmi - Removes one or more images.
 
 ## SYNOPSIS
-**buildah rmi** *image* ...
+**buildah rmi** [*image*] ...
```

Comment:
I think it's better to do the same change to `rmi.go` and `rm` command.

Reply:
@q384566678 I've updated the rm man page with similar changes in #1050 and touched  up the rmi man page slightly too.  I'm not seeing any changes that are need in the .go code for either rm or rmi though.  If you're still seeing changes that are necessary, please let me know.